Add documentation for send_update_notifications query param for Event…

ramillim opened the pull request:
… DELETE and UPDATE requests.

margotkriete commented on README.md:
```diff
@@ -596,6 +596,10 @@ Requires authentication: Yes
 `PUT /api/v1/organizations/:organization_id/events/:event_id`
 
+### Request Params
+* send_update_notifications: Set this to `false` to skip notifications. Defaults to sending email notifications to attendees when certain fields such as the location and timeslots are updated.
```
nit: would put the param name in code quotes (`) to keep consistent w/rest of docs

kylehg replied:
Can you indicate it defaults to `true`. Also instead of such as can you specify the exact fields? It's `location` and `timeslots` right @jaredmcdonald ?

ramillim commented:
Yes, the EventChangeset looks at location and timeslots.

ramillim commented:
Also discovered that #request-params and #request-params-1 were referenced twice in the table of contents, causing the links to navigate to the wrong sections, so I fixed that and added the 2 new entries to the ToC
